packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_screen.dart

```diff
@@ -52,119 +62,251 @@ class _DeepLinkPageState extends State<DeepLinkPage>
+        if (tableView != TableView.domainView) path,
+        SchemeColumn(),
+        OSColumn(),
+        if (!controller.showSpitScreen) ...[
+          StatusColumn(),
+          NavigationColumn(),
+        ],
       ],
-      dataRowMinHeight: bundleByDomain ? bundledDataRowHeight : null,
-      dataRowMaxHeight: bundleByDomain ? bundledDataRowHeight : null,
-      rows: rows,
+      selectionNotifier: controller.selectedLink,
+      defaultSortColumn: tableView == TableView.pathView ? path : domain,
+      defaultSortDirection: SortDirection.ascending,
+      onItemSelected: (item) => controller.showSpitScreen = true,
     );
   }
+}
+
+class _ValidationDetailScreen extends StatelessWidget {
+  const _ValidationDetailScreen({
+    required this.linkData,
+    required this.tableView,
+    required this.controller,
+  });
 
-  DataRow _buildRow(
-    BuildContext context,
-    LinkData data, {
-    MaterialStateProperty<Color?>? color,
-  }) {
-    return DataRow(
-      color: color,
-      cells: [
-        DataCell(Text(data.os)),
-        DataCell(Text(data.scheme)),
-        DataCell(
+  final LinkData linkData;
+  final TableView tableView;
+  final DeepLinksController controller;
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.symmetric(horizontal: largeSpacing),
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
           Row(
+            mainAxisAlignment: MainAxisAlignment.spaceBetween,
             children: [
-              if (data.domainError)
-                Padding(
-                  padding: const EdgeInsets.only(right: denseSpacing),
-                  child: Icon(
-                    Icons.error,
-                    color: Theme.of(context).colorScheme.error,
-                  ),
-                ),
-              Text(data.domain),
+              const Text('Selected Deep link validation details'),
+              IconButton(
+                onPressed: () => controller.showSpitScreen = false,
+                icon: const Icon(Icons.close),
+              ),
             ],
           ),
-        ),
-        DataCell(
-          Row(
-            children: [
-              if (data.pathError)
-                Padding(
-                  padding: const EdgeInsets.only(right: denseSpacing),
-                  child: Icon(
-                    Icons.error,
-                    color: Theme.of(context).colorScheme.error,
-                  ),
+          Text(
+            'This tool assistants helps you diagnose Universal Links, App Links,'
+            ' and Custom Schemes in your app. Web check are done for the web association'
+            ' file on your website. App checks are done for the intent filters in'
+            ' the manifest and info.plist file, routing issues, URL format, etc.',
+            style: Theme.of(context).textTheme.bodySmall,
+          ),
```

Comment:
Consider adding a `PaddedDivider` widget here before 'Domain check'

Reply:
There is no divider in figma so I will follow that.

Reply:
Okay. Generally speaking though @nancyhu2023, does the design file in Figma use the same color scheme / style practices as the remainder of DevTools? Since the Deep Link tool is part of DevTools, I think it is important that we have some consistency with the design patterns used here and those used throughout the remainder of the Devtools tool suite.